plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeMinioAndHmsConnectorSmokeTest.java

```diff
@@ -241,6 +242,14 @@ public void testSchemaEvolutionOnTableWithColumnInvariant()
         assertQuery("SELECT * FROM " + tableName, "VALUES (1, NULL), (2, 20)");
     }
 
+    @Override
+    @Test
+    @Disabled
+    public void testConcurrentInsertsReconciliationForBlindInserts()
+    {
+        // testConcurrentInsertsReconciliation requires safe writes capability to avoid test flakiness
```

Comment:
Since all the writes are happening in Trino, this should work. The writes on s3 are unsafe if you're also writing from Spark.

Reply:
> Since all the writes are happening in Trino, this should work. 

It works most of the time, but sometimes it fails - it happened in the CI and this is why I've decided to disable the test.

IIRC it was linked to `Transaction log locked...` messages.